src/Sylius/Bundle/CoreBundle/Resources/config/services.xml

```diff
@@ -84,7 +84,7 @@
 
         <service id="sylius.shipping_method_resolver.default" class="Sylius\Component\Core\Resolver\EligibleDefaultShippingMethodResolver">
             <argument type="service" id="sylius.repository.shipping_method" />
-            <argument type="service" id="sylius.shipping_eligibility_checker" />
+            <argument type="service" id="sylius.shipping_method_eligibility_checker" />
```

Comment:
Change of service id is a bc break

Reply:
Hmm. I am not changing any service ids. In this code I am injecting the new `shipping_method_eligibility_checker` instead of the old one. Is that a BC break? It's not that I question your point. I just don't understand why :D

Reply:
I will take another look, as it could just misleading for me :P

Reply:
Thanks :D